package.json

```diff
@@ -1,7 +1,6 @@
 {
-  "name": "strong-mesh-models",
-  "version": "3.0.1",
-  "description": "Loopback models for communicating over REST with strong-pm and strong-scheduler",
+  "name": "strong-mesh-core",
```

Comment:
Changing the package name confuses CI. It either needs to match the repo name or be set explicitly in the job config.